src/server/search/search_family.cc

```
@@ -183,6 +183,36 @@ optional<search::Schema> ParseSchemaOrReply(DocIndex::DataType type, CmdArgParse
 #pragma GCC diagnostic pop
 #endif
 
+std::string_view ParseField(CmdArgParser* parser) {
+  std::string_view field = parser->Next();
+  if (field.front() == '@') {
+    field.remove_prefix(1);  // remove leading @ if exists
+  }
+  return field;
```

parser fails silently, don't call front() unconditionally